sdk/core/Azure.Core/README.md

```diff
@@ -1,45 +1,100 @@
 # Overview 
 
-Azure.Core is a reusable library used to implement .NET Azure SDK components. 
-It contains APIs that make it easy to call Azure service endpoints in a convinient, performant, and consistent manner.
+Azure.Core provides shared primitives, abstractions, and helpers for .NET Azure SDK service client libraries. 
```

Comment:
>... that follow the [Azure SDK Design Guidelines for .NET](https://azuresdkspecs.z5.web.core.windows.net/DotNetSpec.html).

I think linking to the guidelines is probably the best documentation for what is in Azure.Core and why.